### ISSUE TITLE
Failed to download wind data since April

### ISSUE THREAD

**Issue by KoalaPrince** [JetStream96/QSimPlanner#31]
Since 1 April, the QSimPlanner has been failing to downloading wind data, on the Questions & Bug Reports page of QSimPlanner's website, someone mentioned that possible cause is that the NOAA GFS had new variable names, which the QSimPlanner did not update to follow up, making the application failed to download wind data.

**Comment by wxman20** [comment]
Yes KoalaPrince, that was me, LOL. They just need update the program to take into account the GRIB data decoding to adopt the new variable names employed by NOAA GFS that was recently released earlier this month.
